### PR TITLE
SS-1029 running serve in rancher desktop

### PR DESCRIPTION
**Pull request by churnikov**
This PR introduces changes that allow serve to run in the Rancher Desktop

There are a couple of manual changes that are required to do for now for this to work:
- Change path to your studio source code in the `serve/values.yaml`;
- After `helm install serve .` delete all network policies `kubectl get networkpolicies`. Commenting them out didn't work for some reason.

How to set this up

[https://scilifelab.atlassian.net/wiki/spaces/AIportal/pages/2107113501/Setup+Serve+on+Local+Machine#%F0%9F%96%A5%EF%B8%8F-[…]uster-(Rancher-Desktop)](https://scilifelab.atlassian.net/wiki/spaces/AIportal/pages/2107113501/Setup+Serve+on+Local+Machine#%F0%9F%96%A5%EF%B8%8F-%F0%9F%96%A5%EF%B8%8F-%F0%9F%96%A5%EF%B8%8F--Setting-up-a-local-Kubernetes-cluster-(Rancher-Desktop))
https://scilifelab.atlassian.net/wiki/spaces/AIportal/pages/2107113501/Setup+Serve+on+Local+Machine#%F0%9F%A7%AA-Running-Serve-with-a-K8s-cluster-(Rancher-Desktop%2Fk3s).

### CODE REVIEW

**Comment by churnikov** [comment]
As far as I understand it, we are going to need to apply these changes to argo cd

File: serve/templates/studio-settings-configmap.yaml [hunk 1]
```diff
@@ -104,6 +104,8 @@ data:
         'django_filters',
         'django_structlog',
         'tagulous',
+        "crispy_forms",
```
Comment (hamzaimran08) [inline serve/templates/studio-settings-configmap.yaml:107]:
```suggestion
```

File: serve/templates/studio-settings-configmap.yaml [hunk 2]
```diff
@@ -104,6 +104,8 @@ data:
...
+        "crispy_bootstrap5",
```
Comment (hamzaimran08) [inline serve/templates/studio-settings-configmap.yaml:108]:
```suggestion
```

File: apps/volumek8s/values.yaml
Comment (hamzaimran08) [inline apps/volumek8s/values.yaml:1]: I think we need to remove the changes in this file

**Review by hamzaimran08 — APPROVED** [review]

**Pull request merged**